src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Card } from '../../types/Card';
+
+import moviesFromServer from '../../api/movies.json';
+
+const cards: Card[] = moviesFromServer.map(card => ({
+  ...card,
+}));
 
 export const MovieCard: React.FC = () => (
```

Comment:
And you should pass one movie as a prop here
```suggestion
export const MovieCard: React.FC = ({movie}) => (
```